cmd/gen-go-sample/bytes.go

```diff
@@ -36,6 +37,8 @@ type fileInfo struct {
 	fileName string
 	// varName is the name of the local variable to hold the bytes of the file.
 	varName string
+
+	comment string
```

Comment:
add a field comment for `comment`
(also, for this comment and the previous one, use a preceding blank line to separate it visually from the previous field)